Client/Frontend/Widgets/SnackButton.swift

```diff
@@ -54,7 +57,6 @@ class SnackButton: UIButton {
     }
 
     func drawSeparator() {
-        let separator: UIView = .build { $0.backgroundColor = UIColor.legacyTheme.snackbar.border }
```

Comment:
The legacy colors can be cleaned up with the changes you've made here 😄 

```
var snackbar: SnackBarColor { return SnackBarColor() }
```

and related dark and protocol code can be removed I believe

Reply:
I just want to clarify, do you want me to remove snackbar variables, SnackBarColor class from LegacyTheme and LegacyDarkTheme?

Reply:
Yes please! With this PR I believe we shouldn't be using any `SnackBarColor` from those legacy classes